ngraph/test/constant_folding.cpp

```diff
@@ -210,61 +210,6 @@ TEST(constant_folding, constant_unsqueeze)
     ASSERT_TRUE(test::all_close_f(values_in, values_out, MIN_FLOAT_TOLERANCE_BITS));
 }
 
-TEST(constant_folding, constant_reshape)
```

Comment:
Do we have the same test for v1 Reshape?

Reply:
yes